packages/manager/apps/web/client/app/dns-zone/new/dns-zone-new.controller.js

```diff
@@ -75,9 +89,12 @@ export default class newDnsZoneCtrl {
   }
 
   onDnsOrderError(error) {
+    const message = error.message || error?.data?.message;
```

Comment:
**question**: shouldn't we prioritize the error message from `data` over the default one ?